index.js

```diff
@@ -31,6 +31,7 @@ const authors = {
   marijn: 'https://www.patreon.com/marijn',
   mikeal: 'https://www.patreon.com/mikeal',
   mmckegg: 'https://www.patreon.com/MattMcKegg',
+  mntnr: 'https://opencollective.com/maintainer',
```

Comment:
This doesn't seem like a valid npm user: https://www.npmjs.com/~mntnr

Reply:
Huh. I should've caught that, and those should be republished. I'll get to this eventually - for now, removed. Thanks, and sorry.